utils/lm-sensors/Makefile

```diff
@@ -8,15 +8,14 @@
 include $(TOPDIR)/rules.mk
 
 PKG_NAME:=lm-sensors
-PKG_VERSION:=3.3.5
-PKG_RELEASE:=4
-
-PKG_SOURCE_PROTO:=git
-PKG_SOURCE_URL:=https://github.com/groeck/lm-sensors.git
-PKG_SOURCE_VERSION:=f8cdcc35bff0785aecf49d9a8484a71ce3ebee4f
-PKG_SOURCE_SUBDIR:=$(PKG_NAME)-$(PKG_VERSION)
-PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.xz
-PKG_MIRROR_HASH:=892e9e2eed78e27fb7a08e8ce78b1164a3d101ba79a9e04f1c1273940d48cf17
+PKG_VERSION:=3.5.0
+PKG_RELEASE:=1
+
+PKG_SOURCE_URL:=https://github.com/lm-sensors/lm-sensors/archive/V$(subst .,-,$(PKG_VERSION))/lm-sensors
+PKG_SOURCE:=$(PKG_NAME)-$(PKG_VERSION).tar.gz
+PKG_HASH:=f671c1d63a4cd8581b3a4a775fd7864a740b15ad046fe92038bcff5c5134d7e0
+PKG_BUILD_DIR:=$(BUILD_DIR)/$(PKG_NAME)-$(subst .,-,$(PKG_VERSION))
```

Comment:
If you're going to make this substitution twice, then either use a macro or a temporary variable.

Reply:
Done